### ISSUE TITLE
Coverflow runs offscreen

### ISSUE THREAD

**Issue by ttocskcaj** [OpenPolytechnicBITProjectGroup/Musician-Venue_App#39]
The coverflow seems to work great but it's running off the edge of my screen both on my desktop and my phone.
![screenshot_20170812-144951](https://user-images.githubusercontent.com/453862/29237214-c5e62666-7f6d-11e7-8890-a9ca529ef4cb.png)

**Comment by lxdnz254** [comment]
Yeah I haven't quite figured how to implement the css properly to make it stay still. It's all in coverflow.less

**Comment by lxdnz254** [comment]
Css is not my strongest bit

**Comment by lxdnz254** [comment]
Adding to that bug, on small width screens (mobile phones) the search bar input & button disappear, these are both css issues. that will need addressing

**Comment by ttocskcaj** [comment]
Moving the search out of a navbar would fix that

**Comment by lxdnz254** [comment]
Could we implement a directive for the search function that was usable in venues and artist page? Or do we need different functions for each section?
